### PR TITLE
Change the example code to use standard node syntax for requiring sign-addon

### PR DESCRIPTION
**Pull request by Standard8**
Switch the import to require as that what node/javascript uses.

### CODE REVIEW

**Comment by Standard8** [comment]
I've now realised the sign-addon code uses import internally, but I'm not totally sure if that's due to commonjs, grunt, webpack or something else. Switching to the more standard node syntax will probably make it easier for most people to use.

File: README.md [hunk 1]
```diff
@@ -29,7 +29,7 @@ Here is how to retrieve a signed version of an
 [XPI file](https://developer.mozilla.org/en-US/docs/Mozilla/XPI):
 
 ````javascript
-import signAddon from 'sign-addon';
+var signAddon = require('sign-addon').default;
```
Comment (kumar303) [inline README.md]: does that work? I thought the commonJS equivalent was just:
````
var signAddon = require('sign-addon');
````

Comment (kumar303) [inline README.md]: This can be `const signAddon = ...` which is still commonJS compatible and is more accurate since it won't be used as a variable.

Comment (kumar303) [inline README.md]: Bah, nevermind. `const` is not supported in Node 0.12 (which is a Mac system default).

**Comment by kumar303** [comment]
you might as well convert the arrow functions too

File: README.md (hunk 1 above)
Comment (Standard8) [inline README.md]: > does that work? I thought the commonJS equivalent was just:
> 
> var signAddon = require('sign-addon');

I'm using plain nodejs. Without the .default `signAddon` is a object with items `default` and `signAddonAndExit`.

Comment (kumar303) [inline README.md]: oh! I see. I didn't realize this; the [es6 module specs](http://www.2ality.com/2014/09/es6-modules-final.html) claim that naming it `default` provides static analysis advantages.

**Comment by kumar303** [comment]
I'll get in what you have so far. Thanks for the patch!

**Pull request merged**